gulp/tasks/test.js

```diff
@@ -104,9 +105,11 @@ function testAcceptanceBrowser() {
     .once( 'close', function( code ) {
       if ( code ) {
         fancyLog( 'Tox tests exited with code ' + code );
+        cb(); // eslint-disable-line callback-return, inline-comments
         process.exit( 1 );
       }
       fancyLog( 'Tox tests done!' );
+      cb(); // eslint-disable-line callback-return, inline-comments
```

Comment:
What would be wrong with `return cb()` here?

Reply:
I was trying to be consistent. I'm now sure it should return a callback.